src/TAction.cpp

```diff
@@ -233,15 +234,15 @@ void TAction::expandToolbar(TToolBar* pT)
         button->setStyleSheet(css);
 
         /*
- * CHECK: The other expandToolbar(...) has the following in this position:
```

Comment:
Seems neither the old or the new form is right - I wonder if it got reformatted (with different settings) in the past?